scipy/sparse/csgraph/tests/test_pydata_sparse.py

```diff
@@ -0,0 +1,197 @@
+import pytest
+
+import numpy as np
+import scipy.sparse as sp
+import scipy.sparse.csgraph as spgraph
+
+from numpy.testing import assert_equal
+
+try:
+    import sparse
+except Exception:
+    sparse = None
+
+pytestmark = pytest.mark.skipif(sparse is None,
+                                reason="pydata/sparse not installed")
+
+
+msg = "pydata/sparse (0.8) does not implement necessary operations"
```

Comment:
If this is still true, can you update the version number to `0.15.1`?

Reply:
Also in the linalg test file

Reply:
Yes, the message refers to `DOK` format only. I updated it with `0.15.1`.